src/api-service/__app__/onefuzzlib/azure/vmss.py

```diff
@@ -246,6 +247,8 @@ def create_vmss(
     tags: Dict[str, str],
 ) -> Optional[Error]:
 
+    instance_config = InstanceConfig.fetch()
```

Comment:
I would suggest passing the additional tags in the tag parameter instead of fetching the instance config here. 
This will put the tag generation in one place making it easier to understand what tags are being added to the VM

Reply:
I'm not sure I know what you mean? Lower down in the file, I include both the parameter tags and the instance_config tags.